qtism/data/AssessmentItem.php

```diff
@@ -658,6 +659,34 @@ public function getModalFeedbacks()
         return $this->modalFeedbacks;
     }
 
+    public function getResponseValidityConstraints(): ResponseValidityConstraintCollection
+    {
+        $classNames = [
```

Comment:
Can we use some constant here? Maybe those classes are defined in some config?

Reply:
yes, better to push it to config I think as this list potentially can be extended